### DISCUSSION TITLE
Powering down the LMN-03

### DISCUSSION THREAD

**Discussion by parmour** [FundamentalFrequency/.github#84] [General]
Hi,
I have just built my LMN-03 and am getting the hang of using it - I think it's an amazing project.

Wondered if I am right in thinking that you have to access the shutdown from the touchscreen to power off the device gracefully?

Or has anyone worked out a more convenient way?

Thanks in advance,
Phil

**Comment by benpocalypse** [comment]
I know the discussion here has gone rather quiet, but if you're still in the market for an  "easy" way to safely shut down your LMN-3, this page has a pretty good set of examples depending on how involved you want it to be: https://learn.pi-supply.com/make/adding-an-on-off-switch-to-your-raspberry-pi/

I personally just wired up a button to one of the GPIO pins and am running a script to safely shut down the Pi.
